lms/templates/courseware/accordion.html

```diff
@@ -25,7 +27,7 @@
         % for section in chapter['sections']:
         <div class="menu-item ${'active' if 'active' in section and section['active'] else ''} ${'graded'  if 'graded' in section and section['graded'] else ''}">
             <a href="${reverse('courseware_section', args=[course_id, chapter['url_name'], section['url_name']])}">
-                <p>${section['display_name']} ${'<span class="sr"> current section</span>' if 'active' in section and section['active'] else ''}</p>
+                <p>${section['display_name']} ${'<span class="sr"> current section</span>' if 'active' in section and section['active'] else '' | n, unicode}</p>
```

Comment:
The changes here are necessary. Without them, the included html gets escaped and appears as plain text on the accordion, like: 
![screen shot 2016-03-31 at 2 18 29 pm](https://cloud.githubusercontent.com/assets/12545624/14186263/7dc20900-f74b-11e5-8198-8072c110f05b.png)